src/main/java/com/manoelcampos/javadoc/coverage/CoverageDoclet.java

```diff
@@ -148,14 +158,14 @@ public static int optionLength(final String option) {
      * @return the values associated to the option, where the 0th element is the option itself;
      * or an empty array if the option is invalid.
      */
-    public String[] getOptionValues(final String[] optionNames) {
+    public String getOptionValue(final String[] optionNames) {
         for (final String[] optionValues : rootDoc.options()) {
             if (isParameter(optionValues[0], optionNames)) {
-                return optionValues;
+                return optionValues[1];
             }
         }
 
-        return new String[]{};
+        return null;
```

Comment:
To avoid NulPointerException, I prefer returning an empty String instead of null. We can use String.isEmpty to check if there is a value for a given parameter.

Reply:
isn't a problem with the new configuration handling any more